src/nfs/metadata/mod.rs

```diff
@@ -16,8 +16,9 @@
 // relating to use of the SAFE Network Software.
 
 /// FileMetadata
-pub mod file_metadata;
-/// DirectoryKey
-pub mod directory_key;
-/// DirectoryMetadata
-pub mod directory_metadata;
+mod file_metadata;
+/// DirMetadata
+mod dir_metadata;
```

Comment:
each grouping should be alphabetical - so rustfmt does it for imports but we will need to do it manually for types/mod/extern-crates etc (or if there is an option now in rustfmt, pls updated the rustfmt.toml and check it in too).